source/form.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page">
+  <header class="page-header">
+    <div class="page-header__logo logo-block">
+      <a href="index.html" class="logo-block__icon">
+        <img src="img/logo/logo-icon.svg" width="34" height="38"
+          alt="Переход на Главную страницу интернет-магазина Cat Energy.">
+      </a>
+      <a href="index.html" class="logo-block__title logo">
+        <img src="img/logo/logo-title.svg" width="101" height="18"
+          alt="Переход на главную страницу интернет-магазина Cat Energy.">
+      </a>
+    </div>
+    <nav class="page-header__navigation main-navigation">
+      <button class="main-navigation__button" aria-label="Открыть меню"></button>
+      <ul class="main-navigation__list menu">
+        <li class="menu__item"><a href="index.html"
+            aria-label="Перейти на Главную страницу интернет-магазина Cat Energy">главная</a></li>
+        <li class="menu__item"><a href="catalog.html"
+            aria-label="Перейти на страницу Каталог интернет-магазина Cat Energy">каталог продукции</a></li>
+        <li class="menu__item menu__item--current"><a>подбор программы</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="form-main">
+    <section class="form-main__program program">
+      <h1 class="program__title title">Подбор программы</h1>
+      <p class="program__text text">Заполните анкету, и мы подберем
+        программу питания для вашего кота</p>
+      <form action="https://echo.htmlacademy.ru" method="POST" class="program-form form">
+        <fieldset class="form__section">
+          <legend class="visually-hidden">Ваш кот</legend>
+          <ul class="form__list">
+            <li class="form__item">
+              <label class="form__legend" for="name-cat">ИМЯ:</label>
```

Comment:
```suggestion
              <label class="form__label" for="name-cat">ИМЯ:</label>
```
предлагаю использовать слово label, вместо legend, чтобы не путать этот класс с классом на теге legend (хотя у тебя такого класса и нет)